admin/class-paper-presenter.php

```diff
@@ -35,21 +35,24 @@ class WPSEO_Paper_Presenter {
 	 * WPSEO_presenter_paper constructor.
 	 *
 	 * @param string $title     The title of the paper.
-	 * @param string $view_file The path to the view file.
+	 * @param string $view_file The path to the view file. Use the content setting if you do not wish to use a view file.
```

Comment:
```suggestion
	 * @param string $view_file Optional. The path to the view file. Use the content setting if you do not wish to use a view file.
```

If the parameter has become optional I expecte `Optional.` to be added to the description.